KenticoCloudDotNetGenerators.Tests/CodeGeneratorTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.CodeAnalysis;
+using NUnit.Framework;
+using System.IO;
+using System.Linq;
+
+namespace KenticoCloudDotNetGenerators.Tests
+{
+    [TestFixture]
+    public class CodeGeneratorTests
+    {
+        [TestCase]
+        public void IntegrationTest()
```

Comment:
ok, see that now. part of the tests use your project. 

we should use this one: 975bf280-fd91-488c-994c-2f04416e5ee3